tools/accuracy_checker/accuracy_checker/config/config_validator.py

```diff
@@ -568,6 +568,47 @@ def parameters(self):
         return parameters_dict
 
 
+class NormalizationArgsField(BaseField):
+    PRECOMPUTED_MEANS = {
+        'imagenet': (104.00698793, 116.66876762, 122.67891434),
+        'cifar10': (125.307, 122.961, 113.8575),
+    }
+
+    PRECOMPUTED_STDS = {
+        'imagenet': (104.00698793, 116.66876762, 122.67891434),
+        'cifar10': (125.307, 122.961, 113.8575),
+    }
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def validate(self, entry, field_uri=None, fetch_only=False, validation_scheme=None):
+        error_stack = super().validate(entry, field_uri, fetch_only, validation_scheme)
+        if entry is None:
+            return error_stack
+
+        field_uri = field_uri or self.field_uri
+
+        precomputed_args = NormalizationArgsField.PRECOMPUTED_MEANS if 'mean' in field_uri else (
```

Comment:
precomputed values can be one from __init__ args, no need to use field_uri as hint

Reply:
Added